servers/rendering_server.cpp

```diff
@@ -2204,6 +2204,23 @@ void RenderingServer::fix_surface_compatibility(SurfaceData &p_surface, const St
 }
 #endif
 
+void RenderingServer::get_argument_options(const StringName &p_function, int p_idx, List<String> *r_options) const {
```

Comment:
Is there any danger to wrapping this in an EDITOR macro? In general data should never be read from the RenderingServer, but particularly, ``global_shader_parameter_get_list()`` should never be called outside the editor (it will fail if called from outside the editor).

Reply:
There's no way for `get_argument_options` to be called outside of the editor. In fact, one can easily argue that all of `get_argument_options`'s overrides should be surrounded with #ifdef TOOLS_ENABLED, but it's not been done yet